connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -237,7 +371,30 @@ public Response resetConnectorActiveTopics(final @PathParam("connector") String
 
     @PUT
     @Path("/{connector}/config")
-    @Operation(summary = "Create or reconfigure the specified connector")
+    @Operation(
+            summary = "Create or reconfigure the specified connector",
+            parameters = {
+                    @Parameter(in = ParameterIn.QUERY, hidden = true, schema = @Schema(implementation = BooleanSchema.class), name = "forward")
+            },
+            requestBody = @RequestBody(
+                    content = @Content(
+                            mediaType = "application/json",
+                            schema = @Schema(implementation = MapSchema.class),
+                            additionalPropertiesSchema = @Schema(type = "string")
+                    )
+            )
+    )
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "201",
+                    description = "New connector created successfully",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ConnectorInfo.class))
+            ),
+            @ApiResponse(
+                    responseCode = "409",
+                    description = "Error while creating new connector. Either rabalance is in progress or the connector already exists"
```

Comment:
This message is not correct. Let's use `Error while creating or updating connector. A rebalance is in progress`.